image/templates/helm/stackrox-secured-cluster/templates/collector.yaml

```diff
@@ -154,9 +210,6 @@ spec:
             path: key.pem
           - key: ca.pem
             path: ca.pem
-      - hostPath:
```

Comment:
note: if we do decide to support language-level vulnerabilities in node-scanning, then will will probably need to mount `/` again, as these packages can, in theory, be anywhere

Reply:
Yes that's a concern. The key issue is the fact that the compliance container does long-running mounts. Have we thought about node-scanning in an "on-demand" model, where the node scanning occurs with a short-term mount of the root directory?

Reply:
We could consider that. I'm not familiar with this concept at all, but we can discuss more when/if we need to add language-level vulns